bitsandbytes/diagnostics/cuda.py

```diff
@@ -152,25 +176,41 @@ def print_cuda_diagnostics(cuda_specs: CUDASpecs) -> None:
 def print_cuda_runtime_diagnostics() -> None:
     cudart_paths = list(find_cudart_libraries())
     if not cudart_paths:
-        print("CUDA SETUP: WARNING! CUDA runtime files not found in any environmental path.")
+        print(f"{BNB_BACKEND} SETUP: WARNING! {BNB_BACKEND} runtime files not found in any environmental path.")
```

Comment:
Might as well simplify the messaging a bit.
```suggestion
        print(f"WARNING! {BNB_BACKEND} runtime files not found in any environmental path.")
```